website/.npmrc

```diff
@@ -0,0 +1,2 @@
+@dbt-labs:registry=https://npm.pkg.github.com
+//npm.pkg.github.com/:_authToken=${NPM_TOKEN}
```

Comment:
For my own edification: is this token necessary so that we can bring in a private package? (I assume react-dbt-dag is private).

Reply:
Yes that is correct!